testing/describes.js

```diff
@@ -734,12 +737,12 @@ class AmpFixture {
       Services.resourcesForDoc(ampdoc).ampInitComplete();
       // Ensure cached meta name/content pairs are cleared before each test
       ampdoc.meta_ = null;
+      maybeTrackImpression(win);
```

Comment:
The `trackImpressionPromise` is still declared under the global `self`, but as long as the tests run in order should be good.